### ISSUE TITLE
Cannot open notebook files from repo in Jupyter running in Docker on MacOS

### ISSUE THREAD

**Issue by bayareacoder** [AI4Finance-LLC/FinRL#228]
Opening notebook file using browser URL `http://localhost:8888/lab/workspaces/auto-S/tree/notebooks/multistock_docker.ipynb` generates a permission denied error (so file was found, but no permission). 

Doing `chmod 777` on any existing notebook file still does't work. When checking `ls -l@` it seems there is an extended file attribute `com.apple.macl` set on the `Untitled.pynb` files auto-generated by Jupyter Notebook on the host system, which seems to be some extra file permission. This is not set on the cloned repo files. Is this why the Docker volume cannot access them?

Is anyone running the Docker install on MacOS and if so, how do you open the repo notebooks?

PS
I followed the README Option1 for installation. I believe whatever steps are needed to resolve this should be added to the README.

![Screen Shot 2021-05-18 at 10 54 16 AM](https://user-images.githubusercontent.com/8239955/118717172-91b14700-b7da-11eb-840d-9a0edc9e3327.png)

**Comment by bayareacoder** [comment]
I found the solution to this, so posting this for the benefit of others. The general symptom was an 'Operation Not Permitted' error when doing an `ls` inside the JupyterLab terminal and no files/paths showed up in the JupyterLab sidebar.
It turns out Docker did not have access to the `Documents` folder hierarchy into which I had cloned the repo code.
Giving Docker access to this folder solved it.
Interestingly enough, any files generated by JupyterLab (instead of opening files already there) did get saved, as mentioned in previous post.

![Screen Shot 2021-05-24 at 5 55 38 PM](https://user-images.githubusercontent.com/8239955/119424057-8c139f80-bcb9-11eb-9a4f-f26d92360bad.png)
